internal/provider/data_source_aci_netflow_monitor_policy.go

```diff
@@ -0,0 +1,202 @@
+// Code generated by "gen/generator.go"; DO NOT EDIT.
+// In order to regenerate this file execute `go generate` from the repository root.
+// More details can be found in the [README](https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/README.md).
+
+package provider
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ciscoecosystem/aci-go-client/v2/client"
+
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+)
+
+// Ensure provider defined types fully satisfy framework interfaces.
+var _ datasource.DataSource = &NetflowMonitorPolDataSource{}
+
+func NewNetflowMonitorPolDataSource() datasource.DataSource {
+	return &NetflowMonitorPolDataSource{}
+}
+
+// NetflowMonitorPolDataSource defines the data source implementation.
+type NetflowMonitorPolDataSource struct {
+	client *client.Client
+}
+
+func (d *NetflowMonitorPolDataSource) Metadata(ctx context.Context, req datasource.MetadataRequest, resp *datasource.MetadataResponse) {
+	tflog.Debug(ctx, "Start metadata of datasource: aci_netflow_monitor_policy")
+	resp.TypeName = req.ProviderTypeName + "_netflow_monitor_policy"
+	tflog.Debug(ctx, "End metadata of datasource: aci_netflow_monitor_policy")
+}
+
+func (d *NetflowMonitorPolDataSource) Schema(ctx context.Context, req datasource.SchemaRequest, resp *datasource.SchemaResponse) {
+	tflog.Debug(ctx, "Start schema of datasource: aci_netflow_monitor_policy")
+	resp.Schema = schema.Schema{
+		// This description is used by the documentation generator and the language server.
+		MarkdownDescription: "The netflow_monitor_policy datasource for the 'netflowMonitorPol' class",
+
+		Attributes: map[string]schema.Attribute{
+			"id": schema.StringAttribute{
+				Computed:            true,
+				MarkdownDescription: "The distinguished name (DN) of the Netflow Monitor Policy object.",
+			},
+			"parent_dn": schema.StringAttribute{
+				Optional:            true,
+				MarkdownDescription: "The distinguished name (DN) of the parent object.",
+			},
```

Comment:
Should we not add default on datasource side also?

```suggestion
			"parent_dn": schema.StringAttribute{
				Computed:        true,
				Optional:            true,
				Default:              stringdefault.StaticString("uni/infra"),
				MarkdownDescription: "The distinguished name (DN) of the parent object.",
			},
```

Reply:
I have tried this solution before. But `Default` does not seem to exist in the Schema of Datasource with the new framework. The documentation is very unclear about it. but by the rendering the template of the datasource with `Default`, it gives me an error stating that it does not recognize this feature.